static/index.html

```diff
@@ -63,15 +63,6 @@
 
     <!-- Settings View -->
     <section data-view="settings" id="settings-view" class="hidden">
-      <div id="section-title" class="hidden">
-        <div id="section-title-back-flex"></div>
-        <div id="section-title-container">
-          <img id="section-title-icon" alt="Section Title" src="#" />
-          <span id="section-title-name"></span>
-        </div>
-        <div id="section-title-speech-flex"></div>
-      </div>
-
```

Comment:
Now that this HTML is part of the layout of individual settings screens, everything has been shifted downwards.

![screen shot 2018-08-24 at 12 42 02](https://user-images.githubusercontent.com/552417/44582927-20641700-a79b-11e8-9e5c-354143517d6d.png)

![screen shot 2018-08-24 at 12 41 46](https://user-images.githubusercontent.com/552417/44582939-24903480-a79b-11e8-9574-9da04aab0e56.png)

![screen shot 2018-08-24 at 12 41 38](https://user-images.githubusercontent.com/552417/44582941-28bc5200-a79b-11e8-94da-a55802dd8a45.png)